src/redux/actions/samples/deleteSamples.js

```diff
@@ -51,13 +50,12 @@ const deleteSamples = (
   try {
     Object.entries(projectSamples).forEach(([projectUuid, samplesToDelete]) => {
       const samplesForAProject = getProjectSamples(projects, projectUuid, samples);
-
       const newSample = _.omit(samplesForAProject, samplesToDelete);
-      newSample.ids = _.difference(samplesForAProject.ids, samplesToDelete);
+      newSample.ids = _.difference(samplesForAProject.ids, samplesToDelete.ids);
```

Comment:
what if `samplesForAProject` is empty? What if `samplesToDelete` is empty?

Reply:
Not sure it will be empty - deleteSamples won't be called when there aren't any samples